parity-dev-node/Dockerfile

```diff
@@ -16,7 +16,7 @@ COPY start.sh /start.sh
 RUN echo "" > /parity/password
 RUN chmod +x /start.sh
 
-ENTRYPOINT /start.sh
+ENTRYPOINT [ "/start.sh" ]
```

Comment:
Why did you add the brackets?

Reply:
Because Docker is stupid.  😢 `ENTRYPOINT /start.sh` is notably different from `ENTRYPOINT [ "/start.sh" ]`.  In this particular case, it needs to be the latter format to support passing in additional variables to the script and docker run time.